### PR TITLE
Added 7z format compress/extract

### PR DESCRIPTION
**Pull request by cfmitrah**

### CODE REVIEW

**Comment by zspitzer** [comment]
any plans to update the commons library being used, 
1.9 was released 2014-10-09, 
1.18  was released  2018-08-16

https://commons.apache.org/proper/commons-compress/changes-report.html#a1.9

seems there are a quite a few fixes for the 7z support since v 1.9

File: core/src/main/java/lucee/commons/io/compress/CompressUtil.java [hunk 1]
```diff
@@ -456,40 +506,51 @@ public static void compressTGZ(Resource[] sources, Resource target, int mode) th
...
+		 // write Gzip
+	    InputStream is = null;
+	    OutputStream os = null;
+		RefBoolean isTempFile2 = new RefBooleanImpl();
+		File srcFile = toFile(file, isTempFile2);
+	    is = new FileInputStream(srcFile);
+		try {
+	     	SevenZArchiveEntry entry = out.createArchiveEntry(srcFile, file.getName());
+	        out.putArchiveEntry(entry);
+	        trg = toResource(targetDir, entry);
```
Comment (mircea-botez) [inline core/src/main/java/lucee/commons/io/compress/CompressUtil.java:530]: this is not really working, at least not on windows, I get a really weird path something like E:\test\a.zip\test.txt, test being the source and a.zip being the destination

**Comment by michaeloffner** [comment]
@cfmitrah the code does not work for me, can you please revisit this and redirect it to 6.0

**Comment by michaeloffner** [comment]
@cfmitrah if that still makes sense, please make a new PR to Lucee 6.1

**Pull request closed without merging**